tests/conftest.py

```diff
@@ -7,7 +7,7 @@
 if TYPE_CHECKING:
     from _pytest.fixtures import SubRequest
 
-PROPS = (
+USER_PROPS = (
```

Comment:
Revert this change. I did not ask you to split user and site props. I asked you to test_macos_homebrew only for site_data_dir and site_cache_dir.

Reply:
Reverted.